build.gradle.kts

```diff
@@ -1,32 +1,27 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext {
+    extra.apply {
         // Cannot be called "ndkVersion" as that leads to naming collision
         // Changes to this value must be reflected in `./docker/Dockerfile`
-        ndkVersionShared = '25.2.9519653'
+        set("ndkVersionShared", "25.2.9519653")
+
     }
+
     repositories {
+        gradlePluginPortal()
         google()
-        jcenter()
+        mavenCentral()
     }
```

Comment:
Just to double check: This block needs to be present both here and in settings.gradle.kts?

Reply:
Yes, it still has to be present because the `buildscript` block is used. 
If it's removed the build fails as it cannot resolve those dependencies in the `buildscript`.

<img width="1369" alt="image" src="https://github.com/syncthing/syncthing-android/assets/6529875/e564c2be-66da-46f3-abe4-b0f5743a6e8b">

It should be possible to get rid of this, I'll try to do that in a future PR.

Reply:
Oh no worries, just wanted to make sure this wasn't a simple oversight